README.md

```diff
@@ -60,6 +60,7 @@ Awesome Artificial Intelligence, Machine Learning and Deep Learning as we learn
         - [Reinforcement learning](./details/julia-python-and-r.md#reinforcement-learning)
       - [ML on Code/Programm/Source Code](./ML-on-code-programming-source-code.md)
     - [Programming in Python](Programming-in-Python.md)
+    - [AI in Golang](./details/julia-python-and-r.md#programming-in-golang)
```

Comment:
Happy to have a new category for GoLang!
And also backlink it from the main README page(s)

Reply:
I had already added them in this PR

Reply:
I meant Golang as a sub-topic on its own: 

### Golang
<your links>